content/billing/managing-billing-for-git-large-file-storage/about-billing-for-git-large-file-storage.md

```diff
@@ -24,7 +24,7 @@ shortTitle: About Git LFS billing
 
 ## Purchasing additional storage and bandwidth
 
-Additional storage and bandwidth is offered in a single data pack. One data pack costs {% data variables.large_files.pack_monthly_price %}, and provides a monthly quota of {% data variables.large_files.pack_monthly_bandwidth %} for bandwidth and {% data variables.large_files.pack_monthly_storage %} for storage. You can purchase as many data packs as you need. For example, if you need 150 GB of storage, you'd buy three data packs.
+Additional storage and bandwidth is offered in a single data pack. One data pack costs {% data variables.large_files.pack_monthly_price %}, and provides a monthly quota of {% data variables.large_files.pack_monthly_bandwidth %} for bandwidth and {% data variables.large_files.pack_monthly_storage %} for storage. [You can purchase](/billing/managing-billing-for-git-large-file-storage/upgrading-git-large-file-storage) as many data packs as you need. For example, if you need 150 GB of storage, you'd buy three data packs.
 
 Purchasing data packs for {% data variables.large_files.product_name_short %} is independent of any other paid feature or product on {% data variables.product.product_name %}.
 
```

Comment:
Since we've added a link to "Upgrading..." above, can you remove it from the "Further reading" below? (Also, if you don't mind, you can remove "Viewing your Git Large File storage usage" and "Downgrading...." as well. Our style guide states that any articles in the sidebar shouldn't be repeated in Further reading.)

Reply:
My initial intent was to spell "purchase" out in the link as "upgrading" at the bottom didn't always associate with "purchase". I've updated your changes, added wording about "purchase", let me know if this works.
I removed unnecessary links from `Further reading` section

Reply:
Hack

Reply:
I don't understand

Reply:
Looks great, thanks!